source/material_model/rheology/frank_kamenetskii.cc

```diff
@@ -23,6 +23,7 @@
 #include <aspect/gravity_model/interface.h>
 #include <aspect/geometry_model/interface.h>
 #include <aspect/utilities.h>
+#include "boost/lexical_cast.hpp"
```

Comment:
Good that you found the problem already. Please separate boost headers from aspect headers by an empty line and use `<>`  instead of `""` to search for headers.

```suggestion

#include <boost/lexical_cast.hpp>
```